Fix compability with Ruby 1.8 by Alex Legler <a3li@gentoo.org>. It helps to actually mark the method static.

--- src/compat18.rb
+++ src/compat18.rb
@@ -12,7 +12,7 @@
 end
 
 class Dir
-	def exists?(path)
+	def self.exists?(path)
 		File.directory?(path)
 	end
 end
